quickstarts/blazor-webassembly/config.yml

```diff
@@ -0,0 +1,53 @@
+id: d47d1298-fe2a-437b-91f1-c0316bd274d7
+
+
+slug: blazor-webassembly
+description: |
+  ## Comprehensive monitoring quickstart for Blazor WebAssembly
+
+  As you probably know, Microsoft developed Blazor WebAssembly to build web UI via C#, HTML, and CSS. With our Browser Monitoring agent, you can monitor your Blazor WebAssembly app performance with our quickstart dashboards right out of the box.
+  
+
+  ## Why monitor Blazor WebAssembly?
+  
+  There are a few strong arguments in favour of using Blazor WebAssembly:
+
+  - Monitoring the higher latency and slower applications.
+  - Using New Relic Browser Monitoring agent captures the Page load time, Web transactions and Https response.
+  - Monitoring the application Performance, Errors and Apdex.
+  - Near-native performance.
+  - Debugging and tooling.
+
+  ## What’s included?
+
+  Get these Blazor WebAssembly monitoring features out of the box:
+  - Understand the health of your system with dashboards that show throughput and error rate data, logs, web transaction time, page views, page load time, and more. 
+  - Get alerts when your application suffers critical failures and errors.
+  - Monitor scripts and functions.
+  - Monitor web transactions.
+
+summary: | 
+  Instrument and monitor your Blazor WebAssembly application with our Browser Monitoring agent. Analyze your performance data right out of the box.
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: Blazor WebAssembly
+documentation:
+  - name: Blazor WebAssembly installation docs
+    description: |
+      Our Browser Monitoring agent can be used to instrument and monitor your Blazor WebAssembly application. Perform an immediate analysis of your performance statistics. 
+    url: >- 
+      https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/browser-monitoring-integrations/blazor-webassembly/
+keywords:  
+
```

Comment:
Let's get rid of the whitespace here, just to ensure that the yaml is parsed correctly
```suggestion
```